cli/stdio.go

```
@@ -100,46 +98,41 @@ func initTheme() {
 }
 
 // Throw an error, show help and die if a different number of arguments than specified is provided
-func requiredArgumentsNumber(ctx *cli.Context, number int) {
-	if ctx.Args().Len() != number {
-		printError("Wrong number of arguments provided. Provided " + pterm.Bold.Sprintf("%d", ctx.Args().Len()) + ", required: " + pterm.Bold.Sprintf("%d", number))
-		cli.ShowSubcommandHelp(ctx)
+func requiredArgumentsNumber(cmd *cli.Command, number int) {
+	if cmd.Args().Len() != number {
+		printError("Wrong number of arguments provided. Provided " + pterm.Bold.Sprintf("%d", cmd.Args().Len()) + ", required: " + pterm.Bold.Sprintf("%d", number))
+		cli.ShowSubcommandHelp(cmd)
```

golangci-lint (1.62.2)

Error return value of `cli.ShowSubcommandHelp` is not checked

(errcheck)